DockerfileMakePackage

```diff
@@ -37,15 +37,19 @@ RUN if [ "$ov_use_binary" == "1" ] ; then true ; else exit 0 ; fi ; cp -v /opt/i
 RUN if [ "$ov_use_binary" == "1" ] ; then true ; else exit 0 ; fi ; rm -vrf /ovms_release/deps/*-devel-*
 
 RUN find /ovms/bazel-out/k8-*/bin -iname '*.so*' -exec cp -v {} /ovms_release/lib/ \;
+RUN cd /ovms_release/lib/ ; rm -f libazurestorage.so.* ; ln -s libazurestorage.so libazurestorage.so.7 ;ln -s libazurestorage.so libazurestorage.so.7.5
```

Comment:
Why do we remove these libraries? Dont we need it for cloud storages?

Reply:
yes, they are used for azure storage, here a copy the the library is replaced with a symbolic link to reduce the image size